source/jormungandr/tests/ptref_tests.py

```diff
@@ -150,6 +150,21 @@ def test_vj_depth_3(self):
         for vj in vjs:
             is_valid_vehicle_journey(vj, depth_check=3)
 
+    def test_frequency_vj(self):
+        """
+        The response should have 3 optional fields
+            start_time
+            end_time
+            headway_secs
+        """
+        response = self.query_region('lines/line:freq/vehicle_journeys?depth=3')
+        vjs = get_not_null(response, 'vehicle_journeys')
+
+        assert len(vjs) == 1
+        assert vjs[0]['start_time'] == '100000'
+        assert vjs[0]['end_time'] == '110000'
+        assert vjs[0]['headway_secs'] == 600  # a 10 min step
+
```

Comment:
What about a test of added attributs on a normal line (not frequency) ?

Reply:
Why not. I'll add it to be more precise

Reply:
The test was very useful because it showed a malfunction with **headway_secs** value